.eslintrc.json

```diff
@@ -14,7 +14,7 @@
   "rules": {
     "jsx-quotes": ["error", "prefer-double"],
     "compat/compat": ["error"],
-    "react/jsx-handler-names": ["warn"], // maybe we want to do this in the future?
+    "react/jsx-handler-names": ["off"],
```

Comment:
Disabled because we don't really follow the rule anywhere, and following it would conflict with our convention of using `_`-prefixed names for private class methods.